### PR TITLE
JENKINS-47888 make run button visible on the PR tab

### PR DESCRIPTION
**Pull request by NicuPascu**
Make the run button visible on the PR tab (it was white against the white background making it invisible)

See [JENKINS-47888](https://issues.jenkins-ci.org/browse/JENKINS-47888).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by NicuPascu** [comment]
Before:
run-button-pr-tab

After:
run-button-pr-tab-fixed

File: blueocean-core-js/src/less/run-button.less [hunk 1]
```diff
@@ -37,6 +37,14 @@
                 position: absolute;
                 top: 0;
                 left: 0;
+
+                .ParametersRunButton & {
+                    fill: rgba(53, 64, 82, 0.25);
```
Comment (sophistifunk) [inline blueocean-core-js/src/less/run-button.less]: Where do these values come from?

**Comment by sophistifunk** [comment]
LGTM 🐝

File: blueocean-core-js/src/less/run-button.less (hunk 1 above)
Comment (NicuPascu) [reply blueocean-core-js/src/less/run-button.less]: I added the same colours as the history button that is beside it, it gets them from here https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/less/history-button.less#L14

Comment (sophistifunk) [reply blueocean-core-js/src/less/run-button.less]: 👍

**Pull request merged**